Content.Client/MassMedia/Ui/NewsReadMenu.xaml.cs

```diff
@@ -0,0 +1,51 @@
+using Content.Client.Message;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+using Content.Shared.MassMedia.Systems;
+
+namespace Content.Client.MassMedia.Ui;
+
+[GenerateTypedNameReferences]
+public sealed partial class NewsReadMenu : DefaultWindow
+{
+    [Dependency] private readonly IEntityManager _entityManager = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+    public event Action? NextButtonPressed;
+    public event Action? PastButtonPressed;
+
+    public NewsReadMenu(string name)
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+
+        if (Window != null)
+            Window.Title = name;
+
+        Next.OnPressed += _ => NextButtonPressed?.Invoke();
+        Past.OnPressed += _ => PastButtonPressed?.Invoke();
+    }
+
+    public void UpdateUI(NewsArticle article, int targetNum, int totalNum)
+    {
+        if (totalNum != -1)
+        {
+            PageName.Text = article.Name;
+            PageText.SetMarkup(article.Content);
+
+            PageNum.Text = targetNum.ToString() + "/" + totalNum.ToString();
+
+            string shareTime = article.ShareTime.ToString("hh\\:mm\\:ss");
+            ShareTime.SetMarkup(Loc.GetString("news-read-ui-time-prefix-text") + " " + shareTime);
+        }
+        else
+        {
+            PageName.Text = Loc.GetString("news-read-ui-not-found-text");
+            PageText.SetMarkup("");
```

Comment:
string.Empty

Reply:
changed the code a little, now here the text becomes invisible